src/util.ts

```diff
@@ -1,41 +1,53 @@
-import * as shelljs from 'shelljs'
-import {type} from 'os'
-import * as fs from 'fs'
-import * as fsPromises from 'fs/promises'
+import * as fs from 'node:fs'
+import * as fsPromises from 'node:fs/promises'
+import {createRequire} from 'node:module'
+import {type} from 'node:os'
 import * as path from 'node:path'
 
-export function sortBy<T>(arr: T[], fn: (i: T) => sortBy.Types | sortBy.Types[]): T[] {
-  function compare(a: sortBy.Types | sortBy.Types[], b: sortBy.Types | sortBy.Types[]): number {
-    a = a === undefined ? 0 : a
-    b = b === undefined ? 0 : b
+type Types = boolean | number | string | undefined
 
-    if (Array.isArray(a) && Array.isArray(b)) {
-      if (a.length === 0 && b.length === 0) return 0
-      const diff = compare(a[0], b[0])
-      if (diff !== 0) return diff
-      return compare(a.slice(1), b.slice(1))
-    }
+function compare(a: Types | Types[], b: Types | Types[]): number {
+  a = a === undefined ? 0 : a
```

Comment:
this would mutate the values outside of this function.

you probably want a new const for them